activities/tool-management/planable.md

```diff
@@ -0,0 +1,21 @@
+---
+type: resource
+---
+
+# Planable
+
+Planable is a collaboration tool for social media teams to create, plan, review, approve, and publish content.
+
+## Using Planable
+
+Our Communications Coordinator and Codebase Stewards for Community have accounts and use the tool to plan and publish social media content.
+
+## Planable management
+
+Our Planable account is managed by the Operations Coordinator and Chief Executive.
+
+## Privacy and GDPR
+
+[Planable's Privacy Policy has detailed information on data collection, data management and GDPR](https://planable.io/privacy/#data-we-collect)
```

Comment:
@demkodo Hey, this section doesn't explain anything about what data Planable collects, or how or where it's processed (the stuff the Foundation's liable for under GDPR).

Reply:
The policy is super extensive, and I added everything that touches on what they collect, how the process it etc. It's a lot. That's why I prefer to simply use an short excerpt and point people to the full policy.